### PR TITLE
Bug 1438067 Added important box to egress router section

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1438067

@danwinship I'll ask for some eyes in the BZ.

### CODE REVIEW

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -226,6 +226,15 @@ specified remote server, using a private source IP address that is not used for
 anything else. The service allows pods to talk to servers that are set up
 to only allow access from whitelisted IP addresses.
 
+[IMPORTANT]
+====
+The egress router is not intended for every outgoing connection. Creating large
+numbers of egress routers, for example, creating an egress router for every
+project or application, can push the limits of your network hardware, such as
+the number of local MAC addresses that the network interface can handle before
+falling back to filtering software.
```
Comment (danwinship) [inline admin_guide/managing_networking.adoc]: "to filtering MAC addresses in software"

**Comment by danwinship** [comment]
otherwise LGTM

**Comment by bfallonf** [comment]
Thanks @danwinship . I've updated it as your suggestion.

@adellape @ahardin-rh @bmcelvee @mburke5678 peer review plz?

File: admin_guide/managing_networking.adoc [hunk 2]
```diff
@@ -226,6 +226,15 @@ specified remote server, using a private source IP address that is not used for
...
 to only allow access from whitelisted IP addresses.
 
+[IMPORTANT]
+====
+The egress router is not intended for every outgoing connection. Creating large
+numbers of egress routers, for example, creating an egress router for every
+project or application, can push the limits of your network hardware, such as
+the number of local MAC addresses that the network interface can handle before
+falling back to filtering MAC addresses in software.
+====
```
Comment (mburke5678) [inline admin_guide/managing_networking.adoc]: Seems like a bit of a run-on sentence. Maybe something like:
"Creating large numbers of egress routers, can push the limits of your network hardware. For example, creating an egress router for every project or application could exceed the number of local MAC addresses that the network interface can handle before falling back to filtering MAC addresses in software."

Comment (bfallonf) [inline admin_guide/managing_networking.adoc]: Much better. Thanks @mburke5678 .

**Comment by bfallonf** [comment]
[rev_history]
|xref:../admin_guide/managing_networking.adoc#admin-guide-manage-networking[Managing Networking]
|Added important box to the xref:../admin_guide/managing_networking.adoc#admin-guide-limit-pod-access-egress-router[Using an Egress Router to Allow External Resources to Recognize Pod Traffic] section.
%

**Pull request merged**